impeller/tessellator/tessellator.cc

```diff
@@ -4,10 +4,13 @@
 
 #include "impeller/tessellator/tessellator.h"
 
+#include "flutter/fml/thread_local.h"
 #include "third_party/libtess2/Include/tesselator.h"
 
 namespace impeller {
 
+FML_THREAD_LOCAL std::vector<Point> tls_point_buffer;
```

Comment:
TLS isn't free, do we actually use the tessellator from multiple threads?

Reply:
AFAICT the underlying libtess2 `TESStesselator` held by the `impeller::Tessellator` is not thread safe.

I'd suggest moving the reusable point buffer into the `impeller::Tessellator` and documenting that `impeller::Tessellator` is not thread safe.

Reply:
That's fair, I was trying to be more defensive here but don't need to be. I'll make this just a regular ivar on impeller::Tessellator.